ts/util/parsePhoneNumber.ts

```diff
@@ -0,0 +1,17 @@
+import { instance, PhoneNumberFormat } from './libphonenumberInstance';
+
+export function parsePhoneNumber(
+  phoneNumber: string,
+  options: {
+    regionCode: string;
+  }
+): string {
+  const { regionCode } = options;
+  const parsedNumber = instance.parse(phoneNumber, regionCode);
+
+  if (instance.isValidNumber(parsedNumber)) {
+    return instance.format(parsedNumber, PhoneNumberFormat.E164);
+  }
+
+  return phoneNumber;
+}
```

Comment:
How about creating `ts/types/PhoneNumber` module and then have `PhoneNumber.parse(…)`? I always find I put stuff in `util` when I can’t quite place it elsewhere.